web/public/locales/es.json

```diff
@@ -78,6 +78,46 @@
     "fully-estimated": "Estimado"
   },
   "estimation-card": {
+    "link": "Más información sobre nuestros modelos de estimación",
+    "estimated_time_slicer_average": {
+      "title": "Los datos son estimados",
+      "pill": "Retraso",
+      "body": "Los datos de esta hora aún no se han transmitido. Los valores mostrados son estimaciones y se reemplazarán por mediciones cuando estén disponibles."
+    },
+    "estimated_mode_breakdown": {
+      "title": "Datos parcialmente estimados",
+      "pill": "Datos incompletos",
+      "body": "Los datos de generación de electricidad correspondientes a zona son incompletos.Los valores faltantes se han calculado mediante un modelo de estimación."
+    },
+    "estimated_reconstruct_breakdown": {
+      "title": "Los datos son siempre estimados",
+      "pill": "Datos imprecisos",
+      "body": "Los datos publicados correspondientes a  esta zona sólo proporcionan la generación total. Las fuentes renovables se estiman utilizando datos meteorológicos en tiempo real, mientras que las fuentes fósiles se deducen basándose en los datos históricos de cada fuente de producción."
+    },
+    "estimated_generic_method": {
+      "title": "Los datos son estimados",
+      "pill": "Datos imprecisos",
+      "body": "Los datos publicados para esta zona no están disponible o están incompletos. Los datos mostrados en el mapa han sido estimados mediante nuestros modelos de estimación, pudiendo diferir de los valores reales."
```

Comment:
Last tiny change but otherwise LGTM. 

there is an "s" missing. 

"body": "Los datos publicados para esta zona no están disponibles o están incompletos. Los datos mostrados en el mapa han sido estimados mediante nuestros modelos de estimación, pudiendo diferir de los valores reales."